packages/components/fontawesome/src/icon.tsx

```diff
@@ -27,7 +27,7 @@ export type IconProps = UIIconProps & IconOptions
  * @see Docs https://yamada-ui.com/components/media-and-icons/fontawesome
  */
 export const Icon = forwardRef<IconProps, "svg">(
-  ({ className, icon, size: fontSize, __css, ...rest }, ref) => {
+  ({ className, icon, fontSize, __css, ...rest }, ref) => {
```

Comment:
Please change

```
({ className, icon, fontSize, __css, ...rest }, ref) => {
```

to

```
({ className, icon, __css, ...rest }, ref) => {
```

Reply:
[7b4c209](https://github.com/yamada-ui/yamada-ui/pull/2537/commits/7b4c209fdfbe99dc503db414e1b96089ee83106c)